bom-2.332.x/pom.xml

```diff
@@ -59,6 +62,22 @@
                 <classifier>tests</classifier>
                 <version>1.15.1</version>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>git</artifactId>
+                <version>${git-plugin.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>git</artifactId>
+                <version>${git-plugin.version}</version>
+                <classifier>tests</classifier>
+            </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>git-client</artifactId>
+                <version>3.11.0</version>
```

Comment:
@MarkEWaite I thought you were going to backport the test fixes?

Reply:
I can do that later today.  I had thought that it would be enough to release a new version of the plugin but had failed to consider that the new version updated its minimum Jenkins version.  Will deliver a patch to 3.11.0 that fixes the tests so that we don't need to carry the test exclusions for 2.332.x

Reply:
Release 3.12.2 and updated in fa87c051f8372a61a35d7d503db3124c2f583e7d